.env.example

```
@@ -0,0 +1,18 @@
+# Anthropic API (Default)
+ANTHROPIC_API_KEY=
+ANTHROPIC_BASE_URL=
+
+# OpenAI API (Optional)
+OPENAI_API_KEY=
+OPENAI_BASE_URL=
+
+# Azure OpenAI API (Optional)
+AZURE_OPENAI_API_KEY=
+AZURE_OPENAI_BASE_URL=
+AZURE_OPENAI_API_VERSION=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AZURE_OPENAI_API_VERSION key should go before the AZURE_OPENAI_BASE_URL key

(UnorderedKey)